components/sst_importer/src/util.rs

```diff
@@ -97,7 +97,7 @@ mod tests {
     use std::{fs, path::Path, sync::Arc};
```

Comment:
all std::fs should be replaced with `tikv_util::file_system`?

Reply:
that'll have to wait on followup PRs